C-Plus-Plus/other/Sorting_an_array_of_zeroes_and_ones.cpp

```diff
@@ -0,0 +1,76 @@
+/* Sorting an array of 0's and 1's
+
+Given an array of size n which contains only 0's and 1's
+in any order,we have to modify the array in such a way
+that 0's are on left side and 1's on the right side.
+Test case-1:
+Input-1:
+N = 5
+arr[] = {0, 0, 1, 1, 0}
+Output-1: 0 0 0 1 1
```

Comment:
Remove redundant comments

Reply:
I have added sample input output at the end of code. Should I change it?